scripts/bake-ide-ami.ps1

```diff
@@ -679,7 +679,7 @@ $jsonObject = @"
 
 
         if ( $InstallBaseSoftware ) {
-
+            
```

Comment:
bake-ide-ami.ps1 changes can be deleted.

Reply:
Hi @robe070 removed changes.